src/Package/Impl/DataInspect/VariableViewModel.cs

```diff
@@ -61,8 +61,7 @@ public sealed class VariableViewModel : RSessionDataObject, IIndexedItem {
                 ShowDetailCommandTooltip = Resources.ShowDetailCommandTooltip;
             }
 
-            const ViewerCapabilities tableCaps = (ViewerCapabilities.Table | ViewerCapabilities.List);
-            CanShowOpenCsv = CanShowDetail && result.CanCoerceToDataFrame && _detailsViewer.Capabilities.HasFlag(tableCaps) && result.Length > 0;
+            CanShowOpenCsv = result.CanCoerceToDataFrame && result.Length > 1;
```

Comment:
Can you also add a check for typename "S4" here? Those will usually have a length of 1, but they can still redefine `as.data.frame` - and if they do, we should show export for them.